src/app/public/src/modules/shared/omnibar-adapter.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class StacheOmnibarAdapterService {
+  private static readonly HAS_OMNIBAR_CLASS_NAME: string = 'stache-omnibar-enabled';
+  private element: any = document.querySelector('.sky-omnibar-iframe');
```

Comment:
I know it's a pain, but any use of the DOM should be wrapped in the `StacheWindowRefService` :-/

(Same as for `applyClassToBody` below.)